yarn.lock

```diff
@@ -975,7 +973,7 @@
   resolved "https://registry.yarnpkg.com/@kwsites/promise-deferred/-/promise-deferred-1.1.1.tgz#8ace5259254426ccef57f3175bc64ed7095ed919"
   integrity sha512-GaHYm+c0O9MjZRu0ongGBRbinu8gVAMd2UZjji6jVmqKtZluZnptXGWhz1E8j8D2HJ3f/yMxKAUC0b+57wncIw==
 
-"@lifeomic/attempt@^3.0.0", "@lifeomic/attempt@^3.0.2":
+"@lifeomic/attempt@^3.0.0":
```

Comment:
Why is it still here?

Reply:
Because we still have airnode packages in dependencies (despite using the postinstall trick).